src/components/styles/_neon-border.less

```diff
@@ -0,0 +1,83 @@
+.neon-border-btn {
+  .neon(@col) {
+    color: @black;
+    background-color: @col;
+    background: transparent;
+    box-shadow: 0 0 6px @col, 0 0 6px @col;
```

Comment:
You forgot the inset here in the first box-shadow

Reply:
@shahednasser hey! I am sorry for so many errors, I hope i have fixed all the errors now, have a look and let me know!